plugin-server/src/worker/vm/extensions/posthog.ts

```diff
@@ -65,8 +67,9 @@ export function createPosthog(server: Hub, pluginConfig: PluginConfig): DummyPos
                 team_id: pluginConfig.team_id,
                 uuid: new UUIDT().toString(),
             }
-            await sendEvent(data)
+            await queueEvent(hub, pluginConfig, data, bufferDirectly ? KAFKA_BUFFER : undefined)
```

Comment:
I'm confused - why would we want to send this to the buffer ever? 

Buffer exists for persons-on-events and delaying processing certain events.

Reply:
Of course we would not, but `e2e.test.ts` makes heavy use of `posthog.capture()`, so I didn't want to break out of the pattern there. I also considered only using  `queueEvent()` directly in `e2e.test.ts`, but that'd require refactoring all the test cases.

Reply:
I think this will be cleaner if:
1. `bufferDirectly` -> `topic = defaultTopic`. less abstraction layers
2. Add a comment for the argument saying it's overridden in tests

Note that boolean arguments are evil. https://understandlegacycode.com/blog/what-is-wrong-with-boolean-parameters/

Reply:
Actually removed this, since it makes more sense to test E2E that _regularly_ captured events go via the buffer